### PR TITLE
Update pandora tech radar

### PR DESCRIPTION
**Pull request by islam9**
Update pandora tech radar:

### CODE REVIEW

File: data/2018_09.tsv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PHP5.5	Languages	Old PHP version	-2	HOLD
+PHP7	Languages	New PHP version	1.7	ADOPT
+Python	Languages		1.6	ADOPT
+Go	Languages		1.8	ADOPT
+JavaScript	Languages		1.8	ADOPT
+OpenAPI (Swagger)	Languages		1.8	ADOPT
+Symfony 2	Techniques; Frameworks & Tools		-2	HOLD
+Symfony 3.2	Techniques; Frameworks & Tools		1.7	ADOPT
+Symfony 4.1	Techniques; Frameworks & Tools		1.4	TRIAL
+NODE.JS	Techniques; Frameworks & Tools		1.6	ADOPT
```
Comment (yemexx1) [inline data/2018_09.tsv:10]: Are we still adopting Node JS? I only see minimal usage on our platform.

**Review by williancarminato — APPROVED** [review]
It's a good start :)

**Review by goke-epapa — APPROVED** [review]

File: data/2018_09.tsv [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Puppet	Platforms & Infrastructure		-2	HOLD
+Kubernetes	Platforms & Infrastructure		1.8	ADOPT
+Docker	Platforms & Infrastructure		1.8	ADOPT
+Nginx	Platforms & Infrastructure		1.8	ADOPT
+AWS Lambda	Platforms & Infrastructure		-0.1	ASSESS
```
Comment (yemexx1) [inline data/2018_09.tsv:15]: Will be good to gather feedback from users of Lambda and decide whether we want to move forward with it or not?

File: data/2018_09.tsv [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+AWS S3	Data Mgt		2	ADOPT
+AWS SNS/SQS	Data Mgt		2	ADOPT
+Redis	Data Mgt		2	ADOPT
+Elasticsearch	Data Mgt		2	ADOPT
+RabbitMQ	Data Mgt		1.4	TRIAL
```
Comment (yemexx1) [inline data/2018_09.tsv:20]: RabbitMQ is not used widely as well, maybe assess trial, if there was any?

**Comment by andresdorantes** [comment]
@islam9 is this only about backend chapter technologies?

**Review by wilsont — APPROVED** [review]

**Comment by williancarminato** [comment]
> @islam9 is this only about backend chapter technologies?

@andresdorantes no but I'm merging this one and you can create one with the frontend technologies :)

**Pull request merged**